src/themes/SeaBreezeTheme/Resources/views/modules/ZikulaSearchModule/Block/search.html.twig

```diff
@@ -0,0 +1,18 @@
+<form id="theme_search" method="post" action="{{ path('search_user_search') }}{# @todo probably an incorrect path #}">
+    <div>
+        <input id="block_search_q" type="search" name="q" size="20" maxlength="255" results="10" autosave="Search" class="theme_search_input" />
+        {% if vars.displaySearchBtn == 1 %}
+        <input class="theme_search_button" type="submit" value="{gt text="Search" domain='zikula'}" />
+        {% endif %}
+        <div style="display: none;">
+            {% for plugin, plugin_option in plugin_options %}
+            {{ plugin_option }}
+            {% endfor %}
+        </div>
+        {searchvartofieldnames data=$modvars.ZikulaSearchModule prefix="modvar" assign="modvariables"}
+        {% for name, value in modvariables %}
+        <input type="hidden" name="{$name|safetext}" value="{$value|safetext}" />
```

Comment:
all three of the items you just referenced are in template overrides and will not be used until the module itself is updated to use a twig template. so, I feel we can merge this.